src/resources/items/ItemForm/index.tsx

```diff
@@ -116,24 +99,13 @@ export default function ItemForm({ isEdit }: FormProps): React.ReactElement {
       <Typography variant='h5' fontWeight='bold' sx={{ padding: '0 15px' }}>
         <constants.ICON_ITEM /> {pageTitle}
       </Typography>
-      <TabbedForm
+      <SimpleForm
         warnWhenUnsavedChanges
         resolver={yupResolver(schema)}
         defaultValues={defaultValues}
         toolbar={<ItemFormToolbar />}>
-        <TabbedForm.Tab label='Core'>
-          <CoreForm batchId={batch?.id} />
-        </TabbedForm.Tab>
-        <TabbedForm.Tab label='Mag tape' icon={<GroupWork />}>
-          <MediaForm type='Tape' source='magTape' />
-        </TabbedForm.Tab>
-        <TabbedForm.Tab label='DVD' icon={<Album />}>
-          <MediaForm type='DVD' source='dvd' />
-        </TabbedForm.Tab>
-        <TabbedForm.Tab label='Paper' icon={<MenuBook />}>
-          <MediaForm type='Paper' source='paper' />
-        </TabbedForm.Tab>
-      </TabbedForm>
+        <CoreForm batchId={batch?.id} />
```

Comment:
now that we have removed the tabs, do we still need the separate CoreForm?

Would it be tidier to bring the component into this module?

Reply:
No, I will move the code to ItemForm component. Thanks!

Reply:
@IanMayo  
We already using CoreForm in ItemShow component so there will be more handling in ItemForm component if we will merge both CoreForm to ItemForm components. Let me know if we still need to do this?

Reply:
Oh, if it's used in two places, let's keep it as a component.